api.planx.uk/s3/controller.ts

```diff
@@ -52,10 +52,6 @@ export const publicDownloadController = async (
 ) => {
   const filePath = buildFilePath(req.params.fileKey, req.params.fileName);
 
-  if (!filePath) {
-    return next({ status: 404, message: "file not found" });
-  }
```

Comment:
It's not possible to hit this block - if we don't have `req.params.fileKey` and `req.params.fileName`, we don't hit this API route.